### PR TITLE
Improve fs.CopyFromOs function

### LINKED ISSUE #200: Improve fs.CopyFromOs function
`fs.CopyFromOs` copies a file from the host to another filesystem. The current implementation looks as follows:

**1. Read the source file**

https://github.com/verless/verless/blob/568bc63459b071b710d8fc496fb1ecf6234f948d/fs/fs.go#L118-L121

**2. Create the destination file**

https://github.com/verless/verless/blob/568bc63459b071b710d8fc496fb1ecf6234f948d/fs/fs.go#L136-L139

**3. Write the source contents into the destination file**

https://github.com/verless/verless/blob/568bc63459b071b710d8fc496fb1ecf6234f948d/fs/fs.go#L140-L142

This works well, but can be optimized to achieve a slightly better performance according to [3 ways to copy files in Go](https://opensource.com/article/18/6/copying-files-go).

**Optimizations**

* Reading the source file should be done using `os.Open` instead of `ioutil.ReadFile`.
* Writing the source contents should be done using ~~`os.Copy`~~ `io.Copy` instead of `io.Writer.Write`.
* Both files should be closed at the end of the loop.

### PR DESCRIPTION
**Pull request by rehanvipin**
Replace use of ioutil with io.Copy  
Fixes #200 

I've followed the steps in the contribution guidelines and tried to keep the code-style as similar to the original as possible, please do let me know if I need to make any changes.

### CODE REVIEW

File: fs/fs.go [hunk 1]
```diff
@@ -114,32 +114,37 @@ func CopyFromOS(targetFs afero.Fs, src, dest string, fileOnly bool) error {
...
+		if err != nil {
 			return err
 		}
+
+		if _, err = io.Copy(destFile, srcFile); err != nil {
+			return err
+		}
+
+		srcFile.Close()
+		destFile.Close()
```
Comment (dominikbraun) [inline fs/fs.go:147]:
```suggestion
		_ = srcFile.Close()
		_ = destFile.Close()
```

**Review by dominikbraun — APPROVED** [review]
This looks good to me, thanks! The only thing I'd suggest is to explicitly ignore the errors returned from `srcFile.Close` and `destFile.Close`.

File: fs/fs.go (hunk 1 above)
Comment (rehanvipin) [reply fs/fs.go]: Thanks for pointing this out  
I was unaware of what `file.Close` returns, but now, after reading some articles/docs I understand it better 👍

**Review by dominikbraun — APPROVED** [review]
LGTM 👍

**Pull request merged**